lifecycle/lifecycle-viewmodel-compose/src/commonMain/kotlin/androidx/lifecycle/viewmodel/compose/SavedStateHandleSaver.kt

```diff
@@ -118,9 +118,10 @@ fun <T : Any> SavedStateHandle.saveable(
     saver: Saver<T, out Any> = autoSaver(),
     init: () -> T,
 ): PropertyDelegateProvider<Any?, ReadOnlyProperty<Any?, T>> =
-    PropertyDelegateProvider { _, property ->
```

Comment:
`qualifiedName` reflection API is not supported yet on web. Please copy `internal expect val <T : Any> KClass<T>.canonicalName` from viewmodel module

Reply:
Fixed